setup.py

```diff
@@ -47,6 +47,6 @@
                 'sphinx_rtd_theme',
                 'sphinx_issues',
             ],
-        'tests': ['backports.tempfile', 'pysoundfile']
+        'tests': ['backports.tempfile', 'pysoundfile', 'pytest', 'pytest-cov', 'tqdm']
```

Comment:
tests were working fine earlier, just curious why we need to add pytest and pytest-cov here?

Reply:
This is to get `pip install -e ".[tests]"` to work (it currently does not work) when developing locally.